### ISSUE TITLE
ERROR with TrensorRT 

### ISSUE THREAD

**Issue by Presenceofthefinest** [NTUYWANG103/APEX_AIMBOT#101]
Hi in advance im an semi-experienced with this so please take that into condsideration and appologies in advance for my ignorance.

I am on windows 11.

I installed Cuda v. 11.8.0 and copied the relevant files of the CuDNN into the respective nvidia gpu toolkit folders. The same goes for the TensorRT 8.5.1.7 files and the python installations. I also added the bin folder to the path and followed the nvidia documentation and completed every step even the optional ones. 

When installing the txt file "requirements" the .whl file throws an error. Easy fix i thought since the file reffered to did not install earlier, therefore i updated the .whl file that would install whilst doing the TensorRT python installation. However i get the error that the wheel is not supported.

Did i not install TensorRT correctly, did i use the wrong versions or am i making another error? I tried fixing this via youtube sesrches, chatgpt, stackoverflow and could not find an answer. I hope you can help me even if my question might be ignorant or stupid.

Thanks if you cared to read

TLDR:

Installed everything according to nvidia documentation - got an error that wheel not supported.  How to fix this?

**Comment by DarylTeh** [comment]
Hi, im assuming u added the .whl files to APEX_AIMBOT-master\tensorrt_python, and updated the path in requirements.txt? You could also try doing 'pip install --upgrade pip setuptools wheel' or 'pip install -U setuptools'
